website/docs/components/logo.mdx

```diff
@@ -0,0 +1,11 @@
+import { StorybookComponent } from "@site/src/components/StorybookComponent";
+
+<StorybookComponent />
```

Comment:
Als je geen `implementations` prop meegeeft toont de implementatie switcher alle implementaties:

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/5381b035-529c-405f-99a5-6826f29d61b5)

Daarnaast is `variant` sinds de update naar Storybook 7 een verplichte property geworden, anders weet ie niet wat de standaard is.

Maar ik zie dat je op regel 5 de default variant wilt tonen, ik denk dat deze regel helemaal weg kan.